yocto/shortlog.sh

```diff
@@ -67,12 +69,13 @@ if [[ ! -z "${diff// }" ]]
 then
     cd "$YOCTO_BUILD_DIR"
     # Get the layers used in the build
-    for layer in `bitbake-layers show-layers | awk 'NR>2 {print $2}'`
+    bitbake-layers show-layers | awk 'NR>2 {print $1 " " $2}' | while read layer_name layer_dir
```

Comment:
Isn't this $2 the "layer" variable?

Reply:
Never mind. I got confused by the github way of compacting the file.

Reply:
The output of the `bitbake-layers show-layers` command is like this:
```
layer                 path                                      priority
==========================================================================
meta-gen20x           /home/mardy/pelux/build/../sources/meta-gen20x  17
meta-ivi              /home/mardy/pelux/build/../sources/meta-ivi/meta-ivi  7
meta-ivi-bsp          /home/mardy/pelux/build/../sources/meta-ivi/meta-ivi-bsp  7
...
```

The previous code was ignoring the first column and putting the second one into the `layer` variable, with the result that the shortlog was not that readable. Here I'm changing it to use the first column as `layer_name` and the second one as `layer_dir` (we still need it, because of a `cd` command in `get_changes()`).